buku

```diff
@@ -618,11 +619,15 @@ class BukuDb:
             return
 
         if not self.json:
-            prompt(results, self.noninteractive)
+            if delete is False:
+                prompt_open(results, self.noninteractive)
```

Comment:
Not sure I understand..
So it would be a regular search with one of the flags -s -S.. and the prompt will as  to open a record or delete them all ?